pkg/rc/farm.go

```diff
@@ -277,12 +278,17 @@ func (rcf *Farm) failsafe(rcFields []rcstore.RCLockResult) {
 		}
 		panic("No RCs are scheduled at all. Create one RC to enable the farm. Panicking to escape a potentially bad situation.")
 	}
+
 	globalReplicaCount := 0
 	for _, rc := range rcFields {
-		if rc.ReplicasDesired > 0 {
+		if rc.ReplicasDesired == nil {
+			panic(fmt.Sprintf("RC %s is missing replicas_desired count", rc.ID))
+		}
+
+		if *rc.ReplicasDesired > 0 {
 			return
 		}
-		globalReplicaCount += rc.ReplicasDesired
+		globalReplicaCount += *rc.ReplicasDesired
 	}
 	if globalReplicaCount == 0 {
```

Comment:
having that there probably makes the intention more understandable, you're right though